core/services/pipeline/task.eth_tx.go

```diff
@@ -159,9 +154,12 @@ func (t *ETHTxTask) Run(ctx context.Context, lggr logger.Logger, vars Vars, inpu
 		SignalCallback:   true,
 	}
 
-	if minOutgoingConfirmations > 0 {
+	if !isMinConfirmationSet {
 		// Store the task run ID, so we can resume the pipeline when tx is confirmed
```

Comment:
nit: `finalized` might be the more accurate term here
```suggestion
		// Store the task run ID, so we can resume the pipeline when tx is finalized
```